### PR TITLE
Possible fix for #354

### PR DESCRIPTION
**Pull request by simonschmidt**
This is more explicit about only filtering out 'Properties' to avoid `DeletionPolicy` getting removed for LogGroup in #354 
Not sure if this has any unintended side effects, but I ran the tests and they passed.

### CODE REVIEW

**Comment by markpeek** [comment]
@simonschmidt I squashed the commits and pushed it here https://github.com/cloudtools/troposphere/commit/77320c91cf86589df0f1e861d1a273d251f4f127. Thank you for the fix.

File: troposphere/__init__.py [hunk 1]
```diff
@@ -160,12 +160,14 @@ def JSONrepr(self):
...
                     "Resource %s required in type %s" % (k, rtype))
         self.validate()
-        # If no other properties are set, only return the Type.
         # Mainly used to not have an empty "Properties".
         if self.properties:
             return self.resource
         elif hasattr(self, 'resource_type'):
-            return {'Type': self.resource_type}
+            return {
+                k: v for k, v in self.resource.items()
```
Comment (oliviervg1) [inline troposphere/__init__.py]: Hi @simonschmidt,

This change has caused Troposphere to throw errors when using python 2.6 as it doesn't support dictionary comprehensions.

Would it possible to fix this per chance?

Thanks!

**Comment by oliviervg1** [comment]
Hi,

I'd like to re-open this issue as it Troposphere now throws an error when using Python 2.6. I've commented on the offending line.

Thanks!

**Comment by markpeek** [comment]
@oliviervg1: I just pushed a change to support 2.6.9.

**Pull request closed without merging**